lib/features/reps_core/reps_core.features.field_base.inc

```diff
@@ -11,6 +11,29 @@
 function reps_core_field_default_field_bases() {
   $field_bases = array();
 
+  // Exported field_base: 'body'.
```

Comment:
Do not export the body in field_base.
This will generate conflict as it is already part of the europa platform

Reply:
ok